### PR TITLE
[FX-4786] Guide for Outlook Email Notifications

### PR DESCRIPTION
**Pull request by wolfgangbecker**
## [FX-4786](https://zombie.atlassian.net/browse/FX-4786)

Added "Configure outlook email notifications" section with a step-by-step guide to create an Outlook email notifications recipe.

### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Name | Link | Comment |

### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Name | Link | Comment |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

[FX-4786]: https://zombie.atlassian.net/browse/FX-4786?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: "Outlook email notifications"
+linkTitle: "Configure outlook email notifications"
+weight: 40
+---
+
+## Configure outlook email notifications for pending pentest findings
```
Comment (mahmoudadly-cobalt) [inline content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md:7]:
```suggestion
## Configure Outlook email notifications for pending pentest findings
```

File: content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+title: "Outlook email notifications"
+linkTitle: "Configure outlook email notifications"
```
Comment (mahmoudadly-cobalt) [inline content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md:3]:
```suggestion
linkTitle: "Configure Outlook email notifications"
```

File: content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+4. Copy and customize the recipe
+  1. Configure which pentest should trigger notifications when findings get created.
+  2. Configure which email address to send the to.
+5. Start the recipe
+
+The following is a step-by-step guide to walk you through the configuration process.
+
+### 1. Create a Cobalt connection.
+
+Find out how to create a Cobalt Connector [here](integrations/integrationbuilder/#connection).
```
Comment (mahmoudadly-cobalt) [inline content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md:24]: These URLs need are broken. They need to be referenced explicitly as a full path. For example:
```suggestion
Find out how to create a Cobalt Connector [here](/integrations/integrationbuilder/#connection).
```

File: content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+### 2. Create an Outlook connection.
+
+The following is a quick guide on how to create a Workato connection for Microsoft Outlook.
+In case you're looking for a more comprehensive and general guide on how to manage connections, check [here](integrations/integrationbuilder/connect-your-applications/).
+
+Go to the `Integration Builder` > `Projects`. Select or create a project on the left-side of the page. Then click on the "Create" button on the top-right and choose the "Connection" option.
+![Create a connection for your project](/integrations/integration_builder/findings_outlook_notifications/create_connection_step1.png "Create a connection for your project")
```
Comment (mahmoudadly-cobalt) [inline content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md:32]: Please use the `img` partial to have a nicer image border and shadow. Text-heavy images don't stand out well in between the doc text. For example:
```suggestion
{{% image src="/integrations/integration_builder/findings_outlook_notifications/create_connection_step1.png" alt="Create a connection for your project" %}}
```

File: content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+A "New Connection" page will show up. Look for "Outlook" and select the corresponding tile. You can optionally use the search field to quickly find the connection.
+![Pick an Outlook connection](/integrations/integration_builder/findings_outlook_notifications/create_connection_step2.png "Pick an Outlook connection")
+
+After selecting the connection type, a form will appear. Provide a name, select a location and click on "Sign in with Windows". If needed, you can configure "Advanced settings" as well.
+![Configure the connection](/integrations/integration_builder/findings_outlook_notifications/create_connection_step3.png "Configure the connection")
```
Comment (mahmoudadly-cobalt) [inline content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md:38]: Please leave an extra empty line between the text and the image. Many images in this page appear inline and break the view.
```suggestion

{{% image src="/integrations/integration_builder/findings_outlook_notifications/create_connection_step3.png" alt="Configure the connection" %}}
```

File: content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
+---
+title: "Outlook email notifications"
```
Comment (mahmoudadly-cobalt) [inline content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md:2]: Looks strange to have the link title different from the page title.
```suggestion
title: "Configure Outlook email notifications"
```

File: content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+  A. [Cobalt > Outlook] When pentest finding moved to Pending Fix, send email notification
+  B. [Cobalt > Outlook] When DAST finding is detected, send email notification
+4. Copy and customize the recipe
+  1. Configure which pentest should trigger notifications when findings get created.
+  2. Configure which email address to send the to.
+5. Start the recipe
+
+The following is a step-by-step guide to walk you through the configuration process.
+
+### 1. Create a Cobalt connection.
```
Comment (mahmoudadly-cobalt) [inline content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md:22]: I am having doubts about this step. We already have the connection creation in step 2, and this link here is showing the whole process and recipe.
Looks like a confusing step, and given the duration of the video, it is a waste of the user's configuration time.

Comment (wolfgangbecker) [reply content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md:22]: The thing is that we need both connections, one for Cobalt and the other one for Outlook. Does it make more sense now?

Comment (mahmoudadly-cobalt) [reply content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md:22]: oh got it. Thank you.

Comment (TaylorBriggs) [reply content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md:22]: [According to Grayson](https://cobalthq.slack.com/archives/C05EYKJ2JKX/p1727104986073709?thread_ts=1727101885.091429&cid=C05EYKJ2JKX), the Cobalt connection should be set up automatically when they initialize the workspace. I think you can start with creating the Outlook connection. That's what I did in the GitHub guide, anyway.

Comment (wolfgangbecker) [reply content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md]: Ok thanks, will remove the Cobalt connection step.

File: content/en/Integrations/IntegrationBuilder/outlook-notifications.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+
+In order to configure Outlook to send emails every time a new finding gets created, we need to do the following:
+
+1. Create an Outlook connection.
+2. Find and select a recipe (2 options):
+    - [Cobalt > Outlook] When pentest finding moved to Pending Fix, send email notification
+    - [Cobalt > Outlook] When DAST finding is detected, send email notification
+3. Copy and customize the recipe
+    1. Configure which pentest should trigger notifications when findings get created.
+    2. Configure which email address to send the to.
```
Comment (TaylorBriggs) [inline content/en/Integrations/IntegrationBuilder/outlook-notifications.md:17]:
```suggestion
    2. Configure which email address to send the notification to.
```

File: content/en/Integrations/IntegrationBuilder/outlook-notifications.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+{{% image src="/integrations/integration_builder/findings_outlook_notifications/create_connection_step4.png" alt="Save the connection" %}}
+
+### 2. Find and select a recipe
+
+Go to the `Integration Builder` > `Library` and select Outlook.
+
+{{% image src="/integrations/integration_builder/findings_outlook_notifications/configure_outlook_notification_step1.png" alt="Outlook tile" %}}
+
+Choose between both options:
```
Comment (TaylorBriggs) [inline content/en/Integrations/IntegrationBuilder/outlook-notifications.md:49]:
```suggestion
Choose one of the options:
```

**Review by TaylorBriggs — APPROVED** [review]
I have some grammar/syntax suggestions but they're not blockers.

I also posted this in the channel -- I think we should get some consensus about how these pages are written and structured since we have multiple authors and our individual styles are varying a bit from one another. I think it's probably worthwhile to have a consistent voice in the documentation.

File: content/en/Integrations/IntegrationBuilder/How to Guides/outlook-notifications.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+
+Go to the `Integration Builder` > `Projects`. Select or create a project on the left-side of the page. Then click on the "Create" button on the top-right and choose the "Connection" option.
+
+{{% image src="/integrations/integration_builder/findings_outlook_notifications/create_connection_step1.png" alt="Create a connection for your project" %}}
+
+A "New Connection" page will show up. Look for "Outlook" and select the corresponding tile. You can optionally use the search field to quickly find the connection.
+
+{{% image src="/integrations/integration_builder/findings_outlook_notifications/create_connection_step2.png" alt="Pick an Outlook connection" %}}
+
+After selecting the connection type, a form will appear. Provide a name, select a location and click on "Sign in with Windows". If needed, you can configure "Advanced settings" as well.
```
Comment (grayson-cobalt) [inline content/en/Integrations/IntegrationBuilder/How to Guides/outlook-notifications.md:39]: > Sign in with Windows

The button in the screenshot says "Sign in with Microsoft" 🤔

Comment (grayson-cobalt) [inline content/en/Integrations/IntegrationBuilder/How to Guides/outlook-notifications.md:39]: > select a location

Is there any particular location we should suggest they put the connection? (typically we put connections in the Cobalt > Connection folder)

Comment (wolfgangbecker) [reply content/en/Integrations/IntegrationBuilder/How to Guides/outlook-notifications.md:39]: 😆 Right, will change it

Comment (wolfgangbecker) [reply content/en/Integrations/IntegrationBuilder/How to Guides/outlook-notifications.md:39]: Yes, I'll add a suggestion to move it into Cobalt > Connection

**Review by mahmoudadly-cobalt — APPROVED** [review]

File: content/en/Integrations/IntegrationBuilder/How to Guides/outlook-notifications.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+
+{{% image src="/integrations/integration_builder/findings_outlook_notifications/configure_outlook_notification_step4.png" alt="Copy Outlook recipe" %}}
+
+#### 3.1 Configure which pentest should trigger notifications when findings get created
+
+Select the Cobalt trigger at the top of the flow diagram. In the sidebar you can choose the pentest for which the recipe will trigger when new findings are created.
+
+{{% image src="/integrations/integration_builder/findings_outlook_notifications/configure_outlook_notification_step5.png" alt="Configure Cobalt connection" %}}
+
+#### 3.2 Configure which email address to send the to
```
Comment (grayson-cobalt) [inline content/en/Integrations/IntegrationBuilder/How to Guides/outlook-notifications.md:77]:
```suggestion
#### 3.2 Configure which email address to send the notification to
```

**Pull request merged**